tests/test_parser/test_patch.py

```
@@ -0,0 +1,106 @@
++++ b/f1
+@@ -1,1 +1,2 @@
+ x
++first
+--- a/f2
++++ b/f2
+@@ -1,1 +1,2 @@
+ y
++second
+"""
+    result = extract_added_lines(diff)
+    added = [l for l in result.split("\n") if l]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)